packages/flatpak.rb

```diff
@@ -3,7 +3,7 @@
 class Flatpak < Package
   description 'Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux.'
   homepage 'https://flatpak.org'
-  @_ver = '1.10.2'
+  @_ver = '1.15.6'
```

Comment:
As we are updating the version, we no longer need the revision tag.

Please change to:
```ruby
version '1.15.6'
license 'LGPL-2.1+'
compatibility 'all'
source_url "https://github.com/flatpak/flatpak/releases/download/#{version}/flatpak-#{version}.tar.xz"
```